http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterToolMonitor.scala

```diff
@@ -36,15 +32,14 @@ object ClusterToolMonitor {
 }
 
 /**
+ * TODO: Is this needed?
```

Comment:
I think this was added to alert on things like Jupyter/welder crashing for a running cluster. Don't think anyone is currently looking at the logs/metrics but I'm wondering if we could use it with Stackdriver alerting..

Reply:
hmmkay...I removed all use of newrelic in this PR, so we should be able to add alerts pretty easily